judo-runtime-core-jsl-itest/src/test/java/hu/blackbelt/judo/runtime/core/jsl/transfer/AutoMappedTransferObjectSingleEntityTest.java

```diff
@@ -329,8 +329,8 @@ void testAutoMappedTransferWithRelationOnDerivedVariations() {
 
         //TODO JNG-4906
         //TODO JNG-4917
```

Comment:
If these issues are resolved, these can be removed.

Reply:
[289922b](https://github.com/BlackBeltTechnology/judo-runtime-core-jsl/pull/205/commits/289922b5c9b80977de3c651e8cfccd8d4bc96fa6)